### DISCUSSION TITLE
Git Tower doesn't support Ghostty

### DISCUSSION THREAD

**Discussion by designbyadrian** [ghostty-org/ghostty#2875] [Feature Requests, Ideas] [answered]
Please see relevant [issue on Warp's repository](https://github.com/warpdotdev/Warp/issues/753).

Tower is a graphical Git client on macOS. It's got shortcuts to open the terminal from within repositories. By default, you can select either macOS Terminal or iTerm2. 

![Image](https://github.com/user-attachments/assets/4c3375ad-c962-49b4-a208-f217fcbd8b57)

Ghostty doesn't appear on this list, and I believe there's something that needs to be done on Ghostty's side, at least from what I've learned from Warp.

Reference:

* [Git Tower](https://www.git-tower.com/mac)

**Comment by mitchellh** [comment, marked as answer]
Based on the provided issue it appears Git Tower needs to do this. There is a suggestion to support making Ghostty the default terminal but that is a separate issue and I believe unrelated to the request here.
